test/module/irohad/torii/query_service_test.cpp

```diff
@@ -56,13 +57,14 @@ class QueryServiceTest : public ::testing::Test {
   std::shared_ptr<MockQueryProcessor> query_processor;
   std::shared_ptr<PbQueryFactory> query_factory;
   std::shared_ptr<PbQueryResponseFactory> query_response_factory;
+
 };
 
 TEST_F(QueryServiceTest, SubscribeQueryProcessorWhenInit) {
   // query service is subscribed to query processor
   EXPECT_CALL(*query_processor, queryNotifier())
       .WillOnce(Return(
-          rxcpp::observable<>::empty<std::shared_ptr<model::QueryResponse>>()));
+          rxcpp::observable<>::empty<PolymorphicWrapper<QueryResponse>>()));
```

Comment:
Note for reviewers: it is ok to have polymorphic wrapper now, it will be substituted in next #995